azext_edge/tests/edge/init/test_base_unit.py

```diff
@@ -338,6 +339,8 @@ def test_register_providers(mocker, registration_state):
         "Microsoft.DeviceRegistry",
         "Microsoft.SecretSyncController",
     ]
+    if input_rp:
+        iot_ops_rps = [input_rp]
     mocked_get_resource_client().providers.list.return_value = [
         {"namespace": namespace, "registrationState": registration_state} for namespace in iot_ops_rps
     ]
```

Comment:
update test `assert len(iot_ops_rps) == len(RP_NAMESPACE_SET)`

Reply:
sigh... this is what happens when you go too fast

Reply:
ikik